test/server/http/admin_test.cc

```diff
@@ -293,5 +294,121 @@ TEST(PrometheusStatsFormatter, FormattedTags) {
   EXPECT_EQ(expected, actual);
 }
 
+TEST(PrometheusStatsFormatter, MetricNameCollison) {
+
+  // Create two counters and two gauges with each pair having the same name,
+  // but having different tag names and values.
+  //`statsAsPrometheus()` should return two implying it found two unique stat names
```

Comment:
Sorry, one more nit that I didn't notice the first few times through: please remove the backticks from `statsAsPrometheus()`.